### PR TITLE
Simplify copy on Install page

### PR DESCRIPTION
**Pull request by palewire**

### CODE REVIEW

**Comment by palewire** [comment]
Once we settle this one, @choldgraf, I think I'm prepared to end my War on Cruft.

File: install.md [hunk 1]
```diff
@@ -1,104 +1,52 @@
 ---
 layout: page
 title: Installing Jupyter
-tagline: Get up and running with JupyterLab or the classic Jupyter Notebook.
+tagline: Get up and running on your computer
 permalink: /install
 ---
 
-## Getting started with JupyterLab
+Project Jupyter's tools are available for installation via the [Python Package Index](https://pypi.org/), the leading repository of software created for the Python programming language. Its offerings can be installed with a variety of package management tools that run via a [command-line interface](https://en.wikipedia.org/wiki/Command-line_interface), including [conda](https://docs.conda.io/), [mamba](https://mamba.readthedocs.io/) and [pipenv](https://pipenv.pypa.io/).
```
[the remark this answers is not in the document]
Comment (choldgraf) [reply install.md:8]: I suggest that we note the existence of many tools, but keep our code instructions to the simplest possible tool, which I believe is pip (especially now that JupyterLab has federated extensions, so there's no javascript stuff needed when you install from pip). Technically I believe pypa recommends pip as the tool for installation anyway: https://packaging.python.org/en/latest/guides/tool-recommendations/#installation-tool-recommendations

File: install.md [hunk 2]
```diff
@@ -1,104 +1,52 @@
...
 
-_The [installation guide](http://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html) contains more detailed instructions_
+Which one you use is up to you. Pipenv is recommended by the [Python Packaging Authority](https://www.pypa.io/). Conda and mamba are preferred by many members of [PyData](https://pydata.org/) and can by acquired via [miniforge](https://github.com/conda-forge/miniforge#mambaforge).
```
Comment (choldgraf) [inline install.md:10]:
```suggestion
This page uses instructions with [pip](https://pip.pypa.io/en/stable/) as it is [the recommended installation tool for Python](https://packaging.python.org/en/latest/guides/tool-recommendations/#installation-tool-recommendations).
If you require _environment management_ as opposed to just installation, you should look into [conda](https://docs.conda.io/), [mamba](https://mamba.readthedocs.io/), and [pipenv](https://pipenv.pypa.io/).

> **note**: If you install JupyterLab with `conda` or `mamba`, we recommend using [the `conda-forge` channel](https://conda-forge.org/).
```

File: install.md [hunk 3]
```diff
@@ -1,104 +1,52 @@
...
 
-### Installation with mamba or conda
+## JupyterLab
 
-JupyterLab can be installed with `mamba` and `conda`:
+Install JupyterLab with one of the following:
```
Comment (choldgraf) [inline install.md:14]:
```suggestion
Install JupyterLab with `pip`:
```

File: install.md [hunk 4]
```diff
@@ -1,104 +1,52 @@
...
 
 ```bash
+pipenv install jupyterlab
```
Comment (choldgraf) [inline install.md:17]:
```suggestion
pip install jupyterlab
```

Comment (choldgraf) [reply install.md:17]: Could we leave the installation steps to just `pip`? I think having multiple installation options will confuse people, assuming that most people that install jupyter by following the instructions on this page are novices who are less-experienced with Python in general. The less information we need to give them to get started, the better.

File: install.md [hunk 5]
```diff
@@ -1,104 +1,52 @@
...
-```
+## Jupyter Notebook
 
-or
+Install the classic Jupyter Notebook with one of the following:
 
 ```bash
+pipenv install notebook
 conda install -c conda-forge notebook
+mamba install -c conda-forge notebook
```
Comment (choldgraf) [inline install.md:35]:
```suggestion
pip install notebook
```

File: install.md [hunk 6]
```diff
@@ -1,104 +1,52 @@
...
-## Getting started with Voilà
+## Voilà
 
-### Installation with mamba or conda
-
-If you use `mamba` or `conda`, you can install it with:
+Install Voilà with one of the following:
 
 ```bash
+pipenv install voila
```
Comment (choldgraf) [inline install.md:49]:
```suggestion
pip install voila
```

File: install.md [hunk 7]
```diff
@@ -1,104 +1,52 @@
...
 mamba install -c conda-forge voila
```
Comment (choldgraf) [inline install.md:50]: Ugh I can't suggest edits for lines that were removed - if you agree w/ the change we can manually remove the extra lines

File: install.md (hunk 2 above)
Comment (palewire) [reply install.md:11]: Should the  note go in the Jupyter Lab section, maybe below the install line, rather that in the top section?

**Comment by palewire** [comment]
A made a bunch of trims in line with your suggestions. Have another look.

**Review by choldgraf — APPROVED** [review]
This looks good to me - I think it's much simpler, but for "total beginner install docs", I think that's a good thing :-)

**Pull request merged**